Evaluate-ModelarDB-Compression/main.py

```diff
@@ -342,12 +342,17 @@ def send_sigint_to_process(process):
                 try:
                     test_data_field_column = test_data.column(field_column_name)
                 except:
-                    # Spaces in the name may have been replaced by underscores.
-                    field_column_name_with_space = field_column_name.replace("_", " ")
-                    test_data_field_column = test_data.column(
-                        field_column_name_with_space
-                    )
-
+                    if field_column_name.__contains__("_"):
```

Comment:
Can a column name that contain spaces never be converted to lower case since? If both transformations can happen to the same column name both transformations should be applied to `field_column_name` instead of only one of them.

Reply:
Removed if statement to handle both cases together.